src/components/MonthRangePicker/MonthRangePicker.test.tsx

```diff
@@ -19,7 +19,7 @@ describe('<MonthRangePicker/> mount', () => {
   const props = {
     onChange: jest.fn(),
     minDate: new Date(2018, 3, 11),
-    maxDate: new Date(2024, 10, 11),
+    maxDate: new Date(2100, 10, 11),
```

Comment:
const getNextYear = () => {
  const currentYear = new Date().getFullYear();
  const nextYear = currentYear + 1;
  return nextYear;
};

const nextYear = getNextYear();

a potem w lnii 22

 maxDate: new Date(nextYear, 10, 11),
 
 ale to takie czepianie się 😄

Reply:
Hahah, zrobimy tak, że w 2100 jak przestanie działać to dam ci stówę.

Reply:
xDDD w takim razie czekam 👴